plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastoreConfig.java

```diff
@@ -44,6 +45,19 @@ public CachingHiveMetastoreConfig setMetastoreCacheTtl(Duration metastoreCacheTt
         return this;
     }
 
+    @NotNull
+    public Duration getStatsCacheTtl()
+    {
+        return statsCacheTtl;
+    }
+
+    @Config("hive.metastore-stats-cache-ttl")
```

Comment:
all these settings should have a common prefix, eg `hive.metastore-cache.stats-ttl` (requires migrating existing ones)